context/app/static/js/components/entity-search/sidebar/ConfigureSearchTable/ConfigureSearchTable.jsx

```diff
@@ -27,29 +27,31 @@ function ConfigureSearchTable({
         <TableHead>
           <TableRow>
             <TableCell />
-            <TableCell>Add Facet</TableCell>
-            <TableCell>Add Column</TableCell>
+            <NoWrapIconTooltipCell tooltipTitle="Selecting this checkbox will add the term as a facet for the search results.">
+              Add Facet
+            </NoWrapIconTooltipCell>
+            <NoWrapIconTooltipCell tooltipTitle="Selecting this checkbox will add the term as a column for the search results table.">
```

Comment:
```suggestion
            <NoWrapIconTooltipCell tooltipTitle="Selecting one of these checkboxes will add the term as a facet for the search results.">
              Add Facet
            </NoWrapIconTooltipCell>
            <NoWrapIconTooltipCell tooltipTitle="Selecting one of these checkboxes will add the term as a column for the search results table.">
```
Tweak wording? Or something more concise. It sounds like there's only one box for each.

Reply:
@tsliaw does this work?

Reply:
@john-conroy Change to "Selecting a checkbox will add the term..."